pkg/api/meta/interfaces.go

```diff
@@ -33,16 +33,14 @@ type VersionInterfaces struct {
 // not support that field (Name, UID, Namespace on lists) will be a no-op and return
 // a default value.
 type Interface interface {
```

Comment:
Probably, although I think at the time it was added the intent was that you would primarily use Interface on the package, thus following the upstream convention.  But now it's much more ObjectInterface than Interface.

----- Original Message -----
> > @@ -33,16 +33,14 @@ type VersionInterfaces struct {
> >  // not support that field (Name, UID, Namespace on lists) will be a no-op
> >  and return
> >  // a default value.
> >  type Interface interface {
> 
> Should this become "ObjectInterface" ?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/3603/files#r23260815